### DISCUSSION TITLE
Github Action always queued status

### DISCUSSION THREAD

**Discussion by liangyuanpeng** [community/community#52146] [Actions]
### Select Topic Area

Product Feedback

### Body

This is my blog repo, Just a hugo blog and the action always queued.   Action does very little, so I don't understand what's going on. 

Thanks any help.

![image](https://user-images.githubusercontent.com/28711504/230335959-2c3798a9-338a-49c1-a9ba-1cf13e5a9ebd.png)

![image](https://user-images.githubusercontent.com/28711504/230336136-64a70e8f-2ef9-4c0e-9abc-bd2b29169f4d.png)

**Comment by airtower-luna** [comment]
Your workflows are configured to use `ubuntu-18.04` runners (the `runs-on` option). It [had been deprecated for a while](https://github.blog/changelog/2022-08-09-github-actions-the-ubuntu-18-04-actions-runner-image-is-being-deprecated-and-will-be-removed-by-12-1-22/) and now isn't available any more. You'll need to update your workflows to run with a current Ubuntu image (`ubuntu-22.04` is the current one).
